Vegeting/Screens/PostDetail/PostDetailViewController.swift

```diff
@@ -83,9 +104,17 @@ final class PostDetailViewController: UIViewController {
         return label
     }()
     
+    private let dividerDot: UILabel = {
+        let label = UILabel()
+        label.text = "·"
+        label.font = .preferredFont(forTextStyle: .subheadline)
+        label.textColor = .gray
+        return label
+    }()
+    
     private let dateLabel: UILabel = {
         let label = UILabel()
-        label.text = "· 12월 19일"
+        label.text = "12월 19일"
         label.font = .preferredFont(forTextStyle: .subheadline)
         label.textColor = .gray
         return label
```

Comment:
```suggestion
        return label
        }
         private let contentTextLabel: UILabel = {
        let label = UILabel()
        label.numberOfLines = 0
        label.text = "저번에 너무 가고싶엇는데~~ 긴급휴무로 못 가서 너무 아쉬웠어여. 유명 맛집이라던데 갔다가 옆에 있는 거북이까지 갔다 오면 좋을 것 같아요!! 날짜는 10월 내에 토요일 중에 가능한 날짜 맞춰서 다녀오려고 해요~ㅎㅎ 같이 맛난 점심먹어용! 나머지는 긴내용을 채우귀 위함위하다다다다 로렌아킬리나 짱~!!!! 너무너무 좋아요. 한국에서도 콘서트 열어주세요,,,,"
            label.textAlignment = .left
        label.font = .preferredFont(forTextStyle: .callout)
        return label
    }()
```
p1;
VerticalAlignmentLabel이 height Anchor가 고정됨에 따라서, alignment를 위해서 만든 코드인데 이제 UILabel로 대처가 가능하기 때문에 데이크가 바꿔주실수 있을까요? 🥹